src/mmds/src/lib.rs

```diff
@@ -90,15 +91,40 @@ fn convert_to_response(request: Request) -> Response {
         );
     }
 
-    if request.method() != Method::Get {
-        let mut response = build_response(
+    respond_to_request(request)
+}
+
+fn respond_to_request(request: Request) -> Response {
```

Comment:
shouldn't we have integration tests for querying the API?
or are they coming in a future PR, once v2 is added?

also unit tests for convert_to_response given an MMDS Get version request

Reply:
Integration tests for querying the API will come in a future PR, once MMDSv2 integration is complete. As for unit tests for `convert_to_response`, these, too, will be part of a later PR that also includes logic for responding to MMDSv2 requests. Does that sound alright to you?

Reply:
okay, that sounds alright